src/libraries/illink-oob.targets

```diff
@@ -1,84 +0,0 @@
-<Project>
```

Comment:
Where these collapsed into a single targets file?

Reply:
The logic of illink-sfx.targets and illink-oob.targets is moved into srx.proj and oob.proj.